RecoTracker/TkSeedGenerator/interface/SeedingOTEDProducer.h

```diff
@@ -0,0 +1,76 @@
+//---------------------------------------------------------------------------
+// class SeedingOTEDProducer
+// author: ebrondol
+// date: July, 2016
+//---------------------------------------------------------------------------
+
+#ifndef RecoTracker_TkSeedGenerator_SeedingOTEDProducer_h
+#define RecoTracker_TkSeedGenerator_SeedingOTEDProducer_h
+
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "DataFormats/BeamSpot/interface/BeamSpot.h"
+#include "DataFormats/TrackerRecHit2D/interface/VectorHit.h"
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "DataFormats/TrajectorySeed/interface/TrajectorySeedCollection.h"
+#include "RecoTracker/MeasurementDet/interface/MeasurementTracker.h"
+#include "TrackingTools/MeasurementDet/interface/LayerMeasurements.h"
+
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+
+class TrajectoryStateUpdator;
+
+class SeedingOTEDProducer : public edm::stream::EDProducer<> {
```

Comment:
usually we don't need "ED" in the derived class name

Reply:
Done